props/blaster.h

```diff
@@ -306,8 +306,19 @@ class Blaster : public PropBase {
 	  hybrid_font.PlayCommon(&SFX_mode);
 	  return;
 	}
-	// TODO: would rather do a Talkie to speak the mode we're in after mode sound
-	beeper.Beep(0.05, 2000.0);
+	if (blaster_mode == MODE_STUN) {
+	  talkie.Say(spSTUN); return;
```

Comment:
Fonts should have the opportunity to override these effects.

Reply:
Are you suggesting we should check for a font version of the spoken words? 
Since there already is a stun.wav that is the blaster sound, something such as "spstun" or similar?

Reply:
How about mdstun, mdauto, mdkill ?

Reply:
Changed to mdstun, mdauto, mdkill. Removed plain "mode" as these replace that with specific sounds for each mode, yes?